devenv/lib/gcloud.py

```diff
@@ -27,17 +27,17 @@ def _install(url: str, sha256: str, into: str) -> None:
     fs.write_script(
         f"{into}/gcloud",
         """#!/bin/sh
-export CLOUDSDK_PYTHON="{root}/python/bin/python3" \
-       PATH="{into}/google-cloud-sdk/bin:${{PATH}}"
+export CLOUDSDK_PYTHON={root}/python/bin/python3 \
+       PATH={into}/google-cloud-sdk/bin:\"${{PATH}}\"
```

Comment:
you don't need backslashes inside a triple quoted string

Reply:
oh nice!